api/src/services/keyService.ts

```diff
@@ -288,8 +288,9 @@ export function createKeyService(
   switch (keySource) {
```

Comment:
This is fine but just noting that createKeyService is exported but only ever used in this module.

Reply:
ah true - I'll fix that